glide.lock

```diff
@@ -393,7 +393,7 @@ imports:
   subpackages:
   - log
 - name: github.com/emicklei/go-restful-swagger12
-  version: 5e28dc7157ebd9247a268c26dd1bee7818aa33f7
+  version: 76d80a40857969dddc1a4c21d1eb82f7ea20e9f5
```

Comment:
I don't know where this sha came from, but it's not in our repo: https://github.com/openshift/emicklei-go-restful-swagger12/commit/76d80a40857969dddc1a4c21d1eb82f7ea20e9f5

Reply:
the old sha was correct

Reply:
I'll clear my glide cache.  I just ran restore.

Reply:
Yeah, that dropped it from glide.